ffi/capi/cpp/examples/fixeddecimal_wasm/Makefile

```diff
@@ -20,16 +20,16 @@ $(ALL_HEADERS):
 a.out: ../../../../../target/debug/libicu_capi_staticlib.a $(ALL_HEADERS) test.cpp
 	$(CXX) -std=c++17 test.cpp ../../../../../target/debug/libicu_capi_staticlib.a -ldl -lpthread -lm -g
 
-../../../../../target/wasm32-unknown-emscripten/release-opt-size/libicu_capi_staticlib.a: FORCE
+../../../../../target/wasm32-unknown-emscripten/release/libicu_capi_staticlib.a: FORCE
 	rustup toolchain install ${ICU4X_NIGHTLY_TOOLCHAIN}
 	rustup component add rust-src --toolchain ${ICU4X_NIGHTLY_TOOLCHAIN}
-	RUSTFLAGS="-Cpanic=abort" cargo +${ICU4X_NIGHTLY_TOOLCHAIN} build --profile=release-opt-size -p icu_capi_staticlib --target wasm32-unknown-emscripten -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort
+	RUSTFLAGS="-Cpanic=abort -Copt-level=s -Clto -Cembed-bitcode" cargo +${ICU4X_NIGHTLY_TOOLCHAIN} build --release -p icu_capi_staticlib --target wasm32-unknown-emscripten -Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort
```

Comment:
Issue: Weird RUSTFLAGS should not touch the top level target directory, _especially_ the `release` profile. OK if they touch `release-opt-size` or some other esoteric profile.

Can you introduce a crate for this one too perhaps?